scripts/package.ts

```diff
@@ -106,6 +107,68 @@ function getVersionSuffix(feature: string, debug: boolean): string {
     return `${debugSuffix}${featureSuffix}${commitSuffix}`
 }
 
+/**
+ * @returns true if curl is available
+ */
+function isCurlAvailable(): boolean {
+    try {
```

Comment:
TARS:<br/>Here's a suggestion for a more secure `downloadFiles` function using the `https` module:
```suggestion
import * as https from 'https'
import * as fs from 'fs'
import * as path from 'path'
import { URL } from 'url'

function downloadFiles(urls: string[], outputDir: string, outputFile: string): Promise<void[]> {
    if (!fs.existsSync(outputDir)) {
        fs.mkdirSync(outputDir, { recursive: true })
    }

    return Promise.all(urls.map(url => {
        return new Promise<void>((resolve, reject) => {
            const parsedUrl = new URL(url)
            const filePath = path.join(outputDir, outputFile || path.basename(parsedUrl.pathname))

            https.get(url, (response) => {
                if (response.statusCode === 200) {
                    const file = fs.createWriteStream(filePath)
                    response.pipe(file)
                    file.on('finish', () => {
                        file.close()
                        resolve()
                    })
                } else {
                    reject(new Error(`Failed to download ${url}: ${response.statusCode}`))
                }
            }).on('error', reject)
        })
    }))
}
```